frontend/src/metabase/containers/SaveQuestionModal.unit.spec.tsx

```diff
@@ -76,7 +79,8 @@ const setup = async (
     setupCollectionsEndpoints(collectionEndpoints);
   } else {
     fetchMock.get("path:/api/collection", TEST_COLLECTIONS);
-    fetchMock.get("path:/api/collection/root", TEST_COLLECTIONS);
+    fetchMock.get("path:/api/collection/root", TEST_COLLECTIONS[0]);
```

Comment:
If `BOBBY_TEST_COLLECTION` deserves an identifier, so does `ROOT_TEST_COLLECTION` ;)

```suggestion
    fetchMock.get("path:/api/collection/root", ROOT_TEST_COLLECTION);
```

Reply:
Fixed, thanks!